snuba/web/views.py

```diff
@@ -44,6 +44,8 @@
 )
 
 
+metrics = MetricsWrapper(environment.metrics, "api")
```

Comment:
This is to maintain compatibility with existing metrics recorded in the web path via https://github.com/getsentry/snuba/blob/7d8f6320520633ad80c71fa424af471535ca8e2a/snuba/web/query.py#L43